libtsuba/include/tsuba/RDG.h

```diff
@@ -26,6 +26,16 @@ class RDGMeta;
 class RDGCore;
 struct PropStorageInfo;
 
+struct KATANA_EXPORT RDGLoadOptions {
+  /// nullopt means the partition associated with the current host's ID will be
+  /// loaded
+  std::optional<uint32_t> partition_to_load{std::nullopt};
```

Comment:
technically, the `std::nullopt` is not required because `std::optional<T>` has a default constructor.

Some more context:
https://github.com/KatanaGraph/katana-enterprise/pull/805#discussion_r567173125

The rule of thumb is to use "uniform initialization", `type name{}`, for non-class types (e.g., int, int*). For class types, you can rely on the default constructor (if appropriate).